docs/jetstream/jetstream.md

```diff
@@ -23,6 +23,13 @@ by contributing
 **experiment configurations** <!-- TODO: linkme -->
 to jetstream-config.
 
+Jetstream is not a monitoring platform,
```

Comment:
I'd recommend framing this as a "how to" i.e.

```
Jetstream does not emit real-time events – if you are looking for monitoring tools, currently your best options are [Grafana etc.]

Reply:
There are some monitoring related docs here: https://docs.telemetry.mozilla.org/datasets/experiment_monitoring.html
Currently, outdated but I'm updating them now.

Reply:
I don't disagree but I don't think we have documentation for monitoring and I don't think we're going to write any yet. The dataset documentation is probably too technical.

I submit that this is a useful clarification to the doc and that we should merge it.